keras/layers/convolutional.py

```diff
@@ -1180,6 +1180,14 @@ def build(self, input_shape):
         if input_shape[channel_axis] is None:
             raise ValueError('The channel dimension of the inputs '
                              'should be defined. Found `None`.')
+
+        if K.backend() == 'theano':
+            if self.depth_multiplier not in (1, 1.):
```

Comment:
`if self.depth_multiplier != 1`

Additionally: please put this exception in the Theano function, in the backend, since it is Theano-specific.

Reply:
I tried, but the problem is in creation of ```depthwise_kernel``` and ```pointwise_kernel```. Their shapes defined as:

```depthwise_kernel_shape = self.kernel_size + (input_dim, self.depth_multiplier)
    pointwise_kernel_shape = (1,) * self.rank + (self.depth_multiplier * input_dim, self.filters)```

Theano can't create variables with shapes containing float values (depth_multiplier).

Reply:
`depth_multiplier` is supposed to be int.